ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -708,40 +708,49 @@ let query_domain_workgroup ~domain =
     workgroup_from_server kdc |> Result.get_ok
   with _ -> raise (Auth_service_error (E_LOOKUP, err_msg))
 
-let config_winbind_damon ~domain ~workgroup ~netbios_name =
+let config_winbind_damon ?(workgroup = "") ?(netbios_name = "") domain =
```

Comment:
These 2 parameters cannot be erased, make them non-optional or add a unit at the end as a parameter

Reply:
I think that has already been updated.